### PR TITLE
0.42.1/GitHub enterprise authentication fix

### PR DESCRIPTION
**Pull request by jpspringall**

### CODE REVIEW

**Comment by jpspringall** [comment]
I understand this cannot me merged as is due to the fact that I have added some md files with further explanation.
But for me at least appeared to resolve #3371

**Comment by jpspringall** [comment]
Proper fix is in #3565

I'll close this PR if it goes in, as i'm referencing it in my fork

File: package.json [hunk 1]
```diff
@@ -14,7 +14,7 @@
 		"tokenInformation",
```
Comment (xrxrjiang) [inline package.json:14]: test

**Comment by alexr00** [comment]
Closing as I don't think this is needed anymore. @jpspringall please correct me if I'm wrong.

**Comment by jpspringall** [comment]
Hey @alexr00 100% correct no longer required with all singing all dancing version :-)

**Pull request closed without merging**